src/components/team/TeamMemberList.tsx

```diff
@@ -34,30 +34,46 @@ const TeamMemberList = ({ teamMembers }: Props) => {
     >
       {teamMembers.map((teamMember, index) => (
         <SwiperSlide key={index}>
```

Comment:
descructure teamMember object to use its values directly: `teamMembers.map(({name, img, ...}, index)`

try to use something more unique as an index. maybe something like: `key={`${name}-${index}`}`